### PR TITLE
Created a directory with an example of a Drupal 6 site 

### PR DESCRIPTION
**Pull request by jordanlgraham**
Gives an example on how to provision from a remote repository and a database stored locally.

### CODE REVIEW

File: existing_drupal_6/existing_drupal_6.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "existing_drupal_6",
+  "description": "The role for my Openexisting_drupal_6 installation",
```
Comment (timodwhit) [inline existing_drupal_6/existing_drupal_6.json]: Looks like a small typo

File: existing_drupal_6/existing_drupal_6.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      "sites": {
+        "existing_drupal_6": {
+          "active": true,
+          "deploy": {
+            "action": ["deploy", "import"]
+          },
+          "drupal": {
+            "settings": {
+              "docroot": "",
+              "db_file": "/assets/existing_drupal_6/existing_drupal_6.mysql"
```
Comment (timodwhit) [inline existing_drupal_6/existing_drupal_6.json]: This path doesn't quite look right. 

If it is on the local machine, it should be coming from `/vagrant/...`

**Comment by jordanlgraham** [comment]
I like to have a separate branch of vampd for each site I'm working on; gives me the ability to provision sites one at a time as necessary.

With this approach I store the sites' databases in /assets/[site_directory], keeping the databases out of version control and allowing me to change branches of vampd without impacting the databases.

**Comment by timodwhit** [comment]
How do you get the db on the machine to be at that path? The reason I ask is because on a clean spinup, `/assets` isn't a shared location, where `/vagrant` is a shared location so if you have a db file at `~/vagrant/vampd/site_db.sql` at `/vagrant/site_db.sql` on the vagrant instance.

**Comment by jordanlgraham** [comment]
Right you are. Answer: I provision based on the 'master' branch of the project, which creates the /assets folder; then I copy and paste the db into the subdirectory of /assets. Not such a good workflow, as I noticed when I destroyed my vbox this morning - and with it, my databases. All backed up elsewhere of course but I'm not loving my approach to db locations.

Problem: where to store site databases such that they persist after the vbox is destroyed, are not version controlled in different branches of vampd and are accessible to vampd on provision.

**Pull request closed without merging**